### PR TITLE
Hw session

### PR DESCRIPTION
**Pull request by suleimanovayev**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop3/controller/InjectInitializer.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package mate.academy.internetshop3.controller;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+
+import mate.academy.internetshop3.lib.Injector;
+import org.apache.log4j.Logger;
+
+public class InjectInitializer implements ServletContextListener {
+    Logger logger = Logger.getLogger(InjectInitializer.class);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop3/controller/InjectInitializer.java:10]: add access modifier

File: src/main/java/mate/academy/internetshop3/controller/RegistrationController.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            throws ServletException, IOException {
+        req.getRequestDispatcher("WEB-INF/views/register.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String name = req.getParameter("user.name");
+        String login = req.getParameter("login");
+        String psw = req.getParameter("psw");
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop3/controller/RegistrationController.java:35]:
```suggestion
        String password = req.getParameter("psw");
```

File: src/main/java/mate/academy/internetshop3/model/User.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+import mate.academy.internetshop3.generator.UserIdGenerator;
+
+public class User {
+
+    private String name;
+    private String surName;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop3/model/User.java:11]:
```suggestion
    private String surname;
```

File: src/main/java/mate/academy/internetshop3/web/MainFilter.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package mate.academy.internetshop3.web;
+
+public class MainFilter {
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop3/web/MainFilter.java]: do we need this filter?

**Review by Sofasmile — COMMENTED** [review]
make small fixes

File: src/main/java/mate/academy/internetshop3/controller/InjectInitializer.java (hunk 1 above)
Comment (suleimanovayev) [reply src/main/java/mate/academy/internetshop3/controller/InjectInitializer.java:10]: ok)

File: src/main/java/mate/academy/internetshop3/web/MainFilter.java (hunk 4 above)
Comment (suleimanovayev) [reply src/main/java/mate/academy/internetshop3/web/MainFilter.java]: no
i forgot to delete

**Pull request closed without merging**